### DISCUSSION TITLE
DSP and number pi

### DISCUSSION THREAD

**Discussion by KonstaMF** [ar1st0crat/NWaves#60] [Ideas]
Hello, I have calculated a more accurate value of Pi, which is 3.24 ((360/55.55555555556)/2=3.24) that is Pi=3.24, not Pi=3.14159........ which participates in the Fourier transform, in particular in the generation of hormonic wave transforms, and is the basis for Digital Signal Processing. Is it possible to develop a universal driver or filter for ASIO and Windows devices with Pi=3.24?

**Comment by damian-666** [comment]
i don't understand is it related to this  https://people.math.ethz.ch/~maechler/R/Rounding.html   

that is 1/18    0.5555555 etc where the problems show.  

but theory aside , to be practical ,  does this manifest in any real issues with the library?    

 have you tried using this value for pi aand can you cite references?  not many audio engines use the DSP or FPGA on ARM and windows is moving to low power and fewer bits for AI and more real time abilities for voice input and such.   XAudio might use them..  for real time filtering after the FFTs make the Kernels. 

from looking at your derivation
and this https://en.wikipedia.org/wiki/Euler%27s_formula  when considering the context of binary floating point and 16 wav and round tripping.  fft to and ifft?   

given that floating point has such flaws that can and do explode in some use cases, and why julia has fractional types.

so im guessing when you mean more accurate, its  with respect to the binary representation, then the base 10 math, assuming the floating floating founding problems and  FFT first principles,  and the 16 bit WAV format .   

i hope to start using fixed point and FPGA or DSP for audio if its going to make a noticeable difference or reduce pops and crackls because bad harmonics means losing detail, windowing and or HF  artifacts and pops.

so confused now but I am well aware of the practical need for non0Euclidian algebra and  the cornea isnt not flat , image processing has had glaring fundamental issues for decades and open new ideas.  Same with   are power spectrums, sound waves fronts, or  and FFTs are flatland.
